common/src/main/java/com/alibaba/nacos/common/http/param/MediaType.java

```diff
@@ -25,7 +25,7 @@ private MediaType() {}
 
     public static final String APPLICATION_ATOM_XML = "application/atom+xml";
 
-    public static final String APPLICATION_FORM_URLENCODED = "application/x-www-form-urlencoded";
+    public static final String APPLICATION_FORM_URLENCODED = "application/x-www-form-urlencoded;charset=utf-8";
```

Comment:
Why add charset in Type? Header.java has add `Accept-Charset` as `UTF-8` default